pkg/gofr/datasource/file/ftp/fs_test.go

```diff
@@ -10,11 +10,18 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/require"
+
 	"github.com/jlaffaye/ftp"
 	"github.com/stretchr/testify/assert"
 	"go.uber.org/mock/gomock"
 )
 
+var (
+	errMockErr  = errors.New("mocked error")
```

Comment:
```suggestion
	errMockSentinel  = errors.New("mocked error")
```

Or anything but "error mock error"